Formula/djl-serving.rb

```diff
@@ -39,6 +39,6 @@ def install
     end
     sleep 30
     cmd = "http://127.0.0.1:#{port}/ping"
-    assert_match "{\n  \"status\": \"Healthy\"\n}\n", shell_output("curl #{cmd}")
+    assert_match "{}\n", shell_output("curl #{cmd}")
```

Comment:
Let's ensure it actually complains 
```suggestion
    assert_match "{}\n", shell_output("curl --fail #{cmd}")
```